src/services/tokenService.ts

```diff
@@ -51,7 +51,7 @@ export const generateCSRFToken = async (cache: Cache): Promise<string> => {
     const hashedToken = await hashToken(csrfToken);
```

Comment:
L51: I don't think you need to await `hashToken()` do you? It's not returning a promise from what I can tell.

Reply:
good catch, I have updated to remove that unnecessary await.